cms/static/js/spec/views/xblock_spec.js

```diff
@@ -95,7 +95,7 @@ define(["jquery", "URI", "common/js/spec_helpers/ajax_helpers", "common/js/compo
                     promise = postXBlockRequest(requests, [
                         ["hash5", { mimetype: "application/javascript", kind: "url", data: missingJavaScriptUrl }]
                     ]);
-                    expect(promise.isRejected()).toBe(true);
+                    expect(promise.state()).toBe("rejected");
```

Comment:
Note to reviewers: it looked like the compatibility library should allow the old syntax, but when running the tests this definitely failed.